### PR TITLE
UPDATES open-issue link for book in README.md

### PR DESCRIPTION
**Pull request by jrichocean**
Updates link for opening issues for the book

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Say "hello" and introduce yourself!
 
 Read our short [Rust and WebAssembly book][book], complete its [tutorials][],
 and compile some Rust into WebAssembly. If you run into a paper cut or
-roadblock, [let us know by filing an issue!][file-issue]
+roadblock, [let us know by filing an issue!][https://github.com/rustwasm/book/issues/new]
```
Comment (CYBAI) [inline README.md]: How about chaning the url for the `file-issue` variable?

**Comment by jrichocean** [comment]
Oh right, good point haha!

**Comment by jrichocean** [comment]
im not sure if you would like to keep the `file-issue` variable in th markdown, i added another var for the book specifically due to the repos splitting and the paragraph referencing filing an issue for the book specifically. The effort is to accompany the [migrate issues](https://github.com/rustwasm/team/issues/197) ticket so that new issues are made in the right repo.

**Comment by mgattozzi** [comment]
This is perfect thanks so much for doing this @jrichocean! :D It's a big help!

**Pull request merged**